health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -89,18 +198,120 @@ const FormulaConfiguration = ({onSelect,  category,  customProps})=>{
+                      style={{height:"50px", fontSize:"20px"}}
+                      title=""
+                      variation="secondary"
+                      isDisabled={isAddNewDisabled}
+                      
+                    />
+                  {showPopUP && <PopUp
+                                    className={"popUpClass"}
+                                    type={"default"}
+                                    heading={t("CONFIRM_TO_DELETE")}
+                                    equalWidthButtons={true}
+                                      children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)